docs/pages/kubernetes-access/guides/standalone-teleport.mdx

```diff
@@ -16,15 +16,20 @@ cluster you want to register. You can use `kubectl config get-contexts` to
 verify that the correct cluster is selected, or `kubectl config use-context
 ${CONTEXT_NAME?}` to switch to cluster `CONTEXT_NAME`.
 
-Next, use [get-kubeconfig.sh](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/get-kubeconfig.sh) to create a `kubeconfig` for Teleport to use.
+Next, use [get-kubeconfig.sh](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/get-kubeconfig.sh) to create a `kubeconfig` for Teleport to use. This will create the `teleport` namespace, the `teleport-sa` serviceaccount, a `teleport-role` clusterrole, and a corresponding `teleport-crb` clusterrolebinding.
 
 You can connect multiple Kubernetes clusters to Teleport from one `kubeconfig` if it contains multiple entries. Use
 [merge-kubeconfigs.sh](https://github.com/gravitational/teleport/blob/master/examples/k8s-auth/merge-kubeconfigs.sh) to combine multiple `kubeconfigs` generated by `get-kubeconfig.sh`.
 
+<Admonition type="warning" title="Warning" >
+    When using `kubeconfig_file`, EKS users may need to replace illegal characters in the `context` names.
```

Comment:
- I think we should move this box to the next section after we introduce `kubeconfig_file`. Otherwise, this may not be clear to users who aren't familiar with that config setting.

- "When using `kubeconfig_file`" makes it sound to me like this setting is optional. I think it would be clearer if the last sentence of the box were first, so readers know more right away why we're adding this admonition. The box would say something like,

```markdown
In EKS `kubeconfig` files, context names typically include the `:` and `@` 
characters, which are invalid in Teleport. EKS users specifying a 
`kubeconfig_file` should make sure the file contains only the following 
characters in `context` names: alphanumeric characters, `.`, `_`, and `-`.
 ```